Core/LocalAdmin.cs

```diff
@@ -181,7 +181,17 @@ internal async Task Start(string[] args)
         {
             await LoadJsonOrTerminate();
 
-            if (DataJson!.EulaAccepted == null)
+            var isEulaNotAccepted = DataJson!.EulaAccepted == null;
+
+            var autoEula = Environment.GetEnvironmentVariable("SLLA_EULA")?.ToUpperInvariant() is "1" or "TRUE"
```

Comment:
1. args.Contains("--eula"**, StringComparer.Ordinal**); - ordinal should be used
2. Value of this is used only once, so there is no need to define a variable. It can be used directly in the if statement.